app/src/main/assets/credits.html

```diff
@@ -2,18 +2,19 @@
 <html>
 <head>
     <meta http-equiv="content-type" content="text/html; charset=utf-8">
+    <script type="text/javascript" src="js/styleContent.js"></script>
     <style>
         body {
           font-family: sans-serif;
         }
         pre {
-          background-color: #eeeeee;
+          background-color: rgba(0, 0, 0, 0);
```

Comment:
isn't this black? correct me if I'm wrong but will the default background color not be black then?

Reply:
@rashiq It makes the background of the pre-formatted content transparent, So that it adapt to its background when the theme is switched.

Reply:
Oh sorry my bad, I didn't see the 0 for the alpha channel.